tmt/steps/provision/__init__.py

```diff
@@ -429,6 +429,9 @@ def push(self, source=None, destination=None, options=None):
             self.debug(f"Push workdir to guest '{self.guest}'.")
         else:
             self.debug(f"Copy '{source}' to '{destination}' on the guest.")
+        # Make sure rsync is present, note: tests can remove it
+        self.debug('Ensure that rsync is installed on the guest.')
+        self.execute('rpm -q rsync || yum install -y rsync')
```

Comment:
Can we have helper function for this? It is duplicated code which harcodes rpm and yum, if we ever support non-rpm distros.

Also - is it OK to use yum? I'm not sure why we need that on rhel-8 and newer...

Reply:
The command is run on the guest so we need `yum` to make it working with both new and older distros.

Reply:
added a helper function to `Guest` class